### PR TITLE
Fixed typos errors - update test.md

### PR DESCRIPTION
**Pull request by EmperialX**
Fixed typos and spelling errors in [test.md](https://github.com/mandiant/flare-floss/blob/master/doc/test.md)

### CODE REVIEW

File: doc/test.md [hunk 1]
```diff
@@ -38,24 +30,16 @@ You can use the following steps to configure your environment for building the b
...
-    change it to `test-decode-rot-13`.
-  - move the file `template.c` to `decode-rot-13.c` and provide your implementation.
-  - update the Makefile in `tests/data/src/Makefile`.
-    add a new line in the first section with the name of your test.
-  - ensure you have the build environment configured, as described in the section
-    "Building Binary Test Cases".
-  - cd to `tests/data/src/decode-rot-13` and run `make all`. confirm the binary runs as expected.
-  - create a new branch named `feature/test-decode-rot-13`,
-    add and commit the Readme, Makefiles, .c source file, compiled binaries, and submit a PR to the [flare-floss-testfiles](https://github.com/mandiant/flare-floss-testfiles) repository.
+  - Decide on a name for your test. Pick something like "decode-rot-13." Follow the examples and stick to this name throughout the test case.
```
Comment (mr-tz) [inline doc/test.md:37]:
```suggestion
  - Decide on a name for your test. Pick something like `decode-rot-13`. Follow the examples and stick to this name throughout the test case.
```

**Comment by EmperialX** [comment]
@mr-tz i remove the commit for test.yml

**Review by mr-tz — APPROVED** [review]
thanks

**Review by mr-tz — CHANGES_REQUESTED** [review]
oh wait, we don't want to remove tests.yml, just the change to it

**Comment by EmperialX** [comment]
@mr-tz done .

**Comment by EmperialX** [comment]
@mr-tz please check now is there any error. thank you

**Comment by symbolicvoid** [comment]
Hey @mr-tz @EmperialX Do we really want to remove the newlines between the text from the file? I found it helpful when editing the docs in pycharm as it displays how the file would look and shortens the file source itself.

File: doc/test.md [hunk 2]
```diff
@@ -38,24 +30,16 @@ You can use the following steps to configure your environment for building the b
...
-  - create a new branch named `feature/test-decode-rot-13`,
-    add and commit the Readme, Makefiles, .c source file, compiled binaries, and submit a PR to the [flare-floss-testfiles](https://github.com/mandiant/flare-floss-testfiles) repository.
+  - Decide on a name for your test. Pick something like `decode-rot-13`. Follow the examples and stick to this name throughout the test case.
+  - Copy the directory `tests/data/src/template` to `tests/data/src/decode-rot-13`.
+  - Update the `test.yml` document to describe the purpose of the test.
+  - Update the Makefile in `tests/data/src/decode-rot-13/Makefile`. You should only need to update the test name in the first line. Change it to `test-decode-rot-13`.
+  - Move the file `template.c` to `decode-rot-13.c` and provide your implementation.
+  - Update the Makefile in `tests/data/src/Makefile`. Add a new line in the first section with the name of your test.
+  - Ensure you have the build environment configured, as described in the section "Building Binary Test Cases."
+  - Move `cd` to `tests/data/src/decode-rot-13` and run `make all`. Confirm the binary runs as expected.
```
Comment (mr-tz) [inline doc/test.md:44]:
```suggestion
  - `cd` to `tests/data/src/decode-rot-13` and run `make all`. Confirm the binary runs as expected.
```

**Comment by mr-tz** [comment]
I like the simplification and fixes here. Plus the rendered output (which most people see) doesn't change. I appreciate you raising your concerns here though.

**Comment by symbolicvoid** [comment]
@mr-tz I like the fixes too. But I just wanted to raise concern about the doc becoming a bit harder to edit. I could convey it better with images. This is how the file looked in pycharm before the removal of newlines:

image_2023-04-05_132337679
Most of the lines are visible and editable without having to scroll sideways.

 This is how it looks after:

image_2023-04-05_132542831

You will have to scroll sideways for most of these lines. This is just an inconvenience when editing this file, as the display remains the same, but I don't see the benefits for doing this either. Apologies if I am making a way bigger issue of it than it has to be, I do appreciate the typo fixes.

**Comment by mr-tz** [comment]
Gotcha, does Pycharm support word wrapping in the editor?

**Comment by symbolicvoid** [comment]
Ah it does support soft wraps. That should be good enough, apologies for this.

**Comment by mr-tz** [comment]
No worries, I think originally this was written exactly for the reason you mentioned.

**Pull request merged**